src/OrchardCore.Themes/TheAdmin/wwwroot/Scripts/TheAdmin.js

```diff
@@ -7029,7 +7029,7 @@ return Popper;
 }));
 //# sourceMappingURL=bootstrap.js.map
 
-function confirmDialog(title, message, handler) {
+function confirmDialog(title, message, okText, cancelText, okCssClass, cancelCssClass, handler) {
     if (title === undefined) {
         title = $('#confirmRemoveModalMetadata').data('title');
```

Comment:
Same question here, why not set a variable instead of duplicating the id everywhere ?